src/Services/ListenTogether/ListenTogether.Application/Interfaces/IRoomGrain.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ListenTogether.Domain;
+using Orleans;
+
+namespace ListenTogether.Application.Interfaces
+{
+    public interface IRoomGrain : IGrainWithStringKey
+    {
+        Task SetRoom(Room room);
+        Task<Room> JoinRoom(string connectionId, string userName);
+        Task<Room> LeaveRoom(string connectionId);
+        Task<Room> UpdatePlayerState(TimeSpan seconds, PlayerState playerState);
+    }
+}
```

Comment:
```suggestion
namespace ListenTogether.Application.Interfaces;

public interface IRoomGrain : IGrainWithStringKey
{
    Task SetRoom(Room room);
    Task<Room> JoinRoom(string connectionId, string userName);
    Task<Room> LeaveRoom(string connectionId);
    Task<Room> UpdatePlayerState(TimeSpan seconds, PlayerState playerState);
}
```

I vote for using file-scoped namespaces, what do you think?